### ISSUE TITLE
Add. 17,215, foll. 36–43 | https://bl.vuexistapps.us/manuscript/905 (Part13)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#715]
### Problems with Manuscript Overview

- In the MS overview, the number of leaves has been mentioned as "9." But since on the web page, the manuscript has been divided into two parts and fol. 35 has its own entry; perhaps the number of leaves should also be reduced. 
- Parchment "Other"?

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- "Nine vellum leaves." since on the web page, the manuscript has been divided into two parts and fol. 35 has its own entry; perhaps the number of leaves should also be reduced. 
- The following sentence appears in the main paragraph of Phys. Des. "There are lacunae after foll. 35, 41, and 42." The thing is, according to the Syriaca division,  that fol. 35 no longer belongs to Add. 17,215, foll. 36–43.
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Resolved
- foll. 35, 41, and 42 changed to foll. 41 and 42
- Add. 17,215, foll. 35—43 changed to 36–43
- Nine vellum leaves changed to "Eight"

Unresolved
- the number of leaves shows 9 at the top. Is this number generated from "measure" under "obj desc"? In that case, I have changed to the folio number there. @wlpotter

**Comment by wlpotter** [comment]
Yes, good catch. This is set in the `measure` object; please update both the `@quantity` to be '8' and the text node to be 8 ff.
